src/app/app.component.html

```diff
@@ -1,11 +1,14 @@
 <app-header></app-header>
 <!-- Content -->
+<app-sub-header></app-sub-header>
 <div class="content" role="main">
+    <app-bugs></app-bugs>
+    <!-- Sub Header -->
+
 
     <!-- Footer -->
-    <app-footer>
-    </app-footer>
+    <app-footer></app-footer>
 
 </div>
 
-<router-outlet></router-outlet>
+<router-outlet></router-outlet>
```

Comment:
Looks like a linting error on app.component.html for no newline at end of file.  Can fix in next PR.